.github/CONTRIBUTING.md

```diff
@@ -381,6 +381,12 @@ When your pull request is created, it is checked by the CLA bot.
 If you have signed the CLA, the status check will be set to `passing`.  Otherwise, it will stay at `pending`.
 Once you sign a CLA, all your existing and future pull requests will have the status check automatically set at `passing`.
 
+## Code of Conduct Enforcement
+
+Reports of abuse will be reviewed by the PS-Committee and if it has been determined that violations of the
+Code of Conduct has occurred, then a temporary ban may be imposed.
```

Comment:
If it is temporary it'd good to point a time - 1 week, 1 month? 
Or better to have a specific policy:
- first 1 week
- the second incident in the same month - 1 month
- more than 10 incidents per year - permanent ban

This would eliminate speculation and misunderstanding.

Reply:
The duration depends on the impact of the infraction.  I can add some text to that effect.